package.json

```diff
@@ -128,14 +129,15 @@
     "hoist-non-react-statics": "^3.3.0",
     "http-status-codes": "^1.3.0",
     "immer": "^1.3.1",
+    "intl-messageformat": "^7.5.4",
```

Comment:
Not sure if this has to be in dependencies... It seems to me that we're using only a type from that https://github.com/gooddata/gooddata-react-components/pull/1265/files#diff-0e397ad7f5ed5798ab35afb129ad30c8R14
Is it really needed or could it be moved to dev deps?

Reply:
moved